metadata-load/src/main/java/uk/ac/ebi/ampt2d/metadata/importer/database/SraObjectsImporterThroughDatabase.java

```diff
@@ -126,130 +157,7 @@ protected Study extractAnalysisFromStudy(StudyType studyType, Study study) {
-
-                case "GRCh37.p4":
-                    accession = "GCA_000001405.5";
-                    break;
-
-                case "GRCh37.p5":
-                    accession = "GCA_000001405.6";
-                    break;
-
-                case "GRCh37.p6":
-                    accession = "GCA_000001405.7";
-                    break;
-
-                case "GRCh37.p7":
-                    accession = "GCA_000001405.8";
-                    break;
-
-                case "GRCh37.p8":
-                    accession = "GCA_000001405.9";
-                    break;
-
-                case "GRCh37.p9":
-                    accession = "GCA_000001405.10";
-                    break;
-
-                case "GRCh37.p10":
-                    accession = "GCA_000001405.11";
-                    break;
-
-                case "GRCh37.p11":
-                    accession = "GCA_000001405.12";
-                    break;
-
-                case "GRCh37.p12":
-                    accession = "GCA_000001405.13";
-                    break;
-
-                case "GRCh37.p13":
-                    accession = "GCA_000001405.14";
-                    break;
-
-                case "GRCh38":
-                    accession = "GCA_000001405.15";
-                    break;
-
-                case "GRCh38.p1":
-                    accession = "GCA_000001405.16";
-                    break;
-
-                case "GRCh38.p2":
-                    accession = "GCA_000001405.17";
-                    break;
-
-                case "GRCh38.p3":
-                    accession = "GCA_000001405.18";
-                    break;
-
-                case "GRCh38.p4":
-                    accession = "GCA_000001405.19";
-                    break;
-
-                case "GRCh38.p5":
-                    accession = "GCA_000001405.20";
-                    break;
-
-                case "GRCh38.p6":
-                    accession = "GCA_000001405.21";
-                    break;
-
-                case "GRCh38.p7":
-                    accession = "GCA_000001405.22";
-                    break;
-
-                case "GRCh38.p8":
-                    accession = "GCA_000001405.23";
-                    break;
-
-                case "GRCh38.p9":
-                    accession = "GCA_000001405.24";
-                    break;
-
-                case "GRCh38.p10":
-                    accession = "GCA_000001405.25";
-                    break;
-
-                case "GRCh38.p11":
-                    accession = "GCA_000001405.26";
-                    break;
-
-                case "GRCh38.p12":
-                    accession = "GCA_000001405.27";
-                    break;
-
-                case "GRCh38.p13":
-                    accession = "GCA_000001405.28";
-                    break;
-
-                default:
-                    throw new Exception("Encountered exception for unknown reference sequence name " + refName);
```

Comment:
This exception still applies, using a Map doesn't prevent the user from requesting an unknown reference. But it should be an `IllegalArgumentException` instead of a completely generic `Exception`.

Reply:
Fixed by @iyangar; in the current implementation `IllegalArgumentException` is thrown